api/services/errorHandlers/signupFailedError.js

```diff
@@ -0,0 +1,9 @@
+const appError = require('./appError');
+
+class signupFailedError extends appError {
+  constructor(message) {
+    super(message || 'Registration failed!', 500);
+  }
+}
```

Comment:
Change the message here. It is not understandable for user, why the registration failed (may be notify him that this is server error)